app/src/main/res/values-sw360dp/dimens.xml

```diff
@@ -55,4 +55,7 @@
     <dimen name="dp_23">23dp</dimen>
     <dimen name="dp_18">18dp</dimen>
     <dimen name="dp_270">270dp</dimen>
+    <dimen name="dp_400">400dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_400` appears to be unused
 Unused resources make applications larger and slow down builds.